pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -64,6 +66,11 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.slack_client = self.runtime.new_slack_client()
         self.slack_client.bind_channel(f'openshift-{self.version}')
 
+        # Imagestream name for Brew builds is 4.y-art-latest
+        # For konflux, it is 4.y-konflux-art-latest
+        self.is_base_name = f'{self.version}-art-latest' if build_system == 'brew'\
+            else f'{self.version}-konflux-art-latest'
```

Comment:
i see this part repeated from `def default_imagestream_base_name` .. can we DRY?

Reply:
Unfortunately, `default_imagestream_base_name` requires a Doozer runtime param to access `build_system`, and we don't have direct access to such an object from pyartcd.